[RFR] Inject emitter

ThieryMichel opened the pull request.

ThieryMichel committed:
allow to inject custom eventEmitter in sg

ThieryMichel committed:
update readme

djhi commented on README.md:
```diff
@@ -215,3 +215,9 @@ During execution handleSql will get called like so
+
+## injecting custom eventEmitter
+
+Sg use an eventEmitter internally to handle take and put effect.
```
s/use/uses

djhi replied:
s/effect/effects/

djhi commented on README.md:
```diff
@@ -215,3 +215,9 @@ During execution handleSql will get called like so
+## injecting custom eventEmitter
+
+Sg use an eventEmitter internally to handle take and put effect.
+It is possible to pass your own eventEmitter to sg. This allow to take event from this event emitter.
```
s/allow/allows/

djhi replied:
s/take event/take events/

sedy-bot committed:
Typo fix s/effect/effects/

As requested by djhi at https://github.com/marmelab/sg/pull/20#discussion_r104871568

sedy-bot committed:
Typo fix s/allow/allows/

As requested by djhi at https://github.com/marmelab/sg/pull/20#discussion_r104871616

sedy-bot committed:
Typo fix s/take event/take events/

As requested by djhi at https://github.com/marmelab/sg/pull/20#discussion_r104872244

jpetitcolas commented on README.md:
````diff
@@ -215,3 +215,9 @@ During execution handleSql will get called like so
 handleSql(['INSERT ... INTO user', userData]);
 ```
+
+## injecting custom eventEmitter
````
s/injecting custom eventEmitter/Injecting Custom EventEmitter/

sedy-bot committed:
Typo fix s/injecting custom eventEmitter/Injecting Custom EventEmitter/

As requested by jpetitcolas at https://github.com/marmelab/sg/pull/20#discussion_r104872781

jpetitcolas commented on README.md:
```diff
@@ -215,3 +215,9 @@ During execution handleSql will get called like so
+
+## injecting custom eventEmitter
+
+Sg use an eventEmitter internally to handle take and put effects.
```
s/use/uses/
s/take/`take`/
s/put/`put`/
